ubuntu-advantage

```diff
@@ -64,6 +64,7 @@ Currently available are:
 - Canonical FIPS 140-2 Non-Certified Module Updates
 - Canonical Livepatch Service (https://www.ubuntu.com/server/livepatch)
 - Canonical Common Criteria EAL2 certification artifacts provisioning
+- Canonical CIS Ubuntu Benchmark Audit tool
```

Comment:
I think you need to rebase on top of master, because the CIS audit tool is there already.

Reply:
Never mind :)